ddl/db_change_test.go

```diff
@@ -336,8 +336,8 @@ type stateCase struct {
 	session            session.Session
 	rawStmt            ast.StmtNode
 	stmt               sqlexec.Statement
-	expectedExecErr    error
-	expectedCompileErr error
+	expectedExecErr    string
```

Comment:
Why  change error to string？

Reply:
In this test case, we actually check the error message by comparing string with `expectedExecErr`, so we can direct store expected string here.